front/app/containers/EventsShowPage/components/EventDescription.tsx

```diff
@@ -22,11 +22,13 @@ const EventDescription = ({ event }: Props) => {
   if (!isNilOrError(event)) {
     return (
       <>
-        <ReadMoreWrapper
-          fontSize="base"
-          contentId="event-description"
-          value={event.attributes?.description_multiloc}
-        />
+        <Box id="e2e-event-description">
```

Comment:
I wonder if it's not better to use `data-cy` attribute, just to make it clear this is needed for an e2e test (even though the id already makes that pretty clear, I guess).

Reply:
Yeah you're right - I'll switch it to a data-cy :) :+1: